tests/urls.py

```diff
@@ -6,10 +6,11 @@
 
 
 urlpatterns = [
+    url(r'^', include('django_freeradius.urls')),
```

Comment:
add the namespace:

`url(r'^', include('django_freeradius.urls', namespace='freeradius')),`

this will mean you will have to update the call to `reverse()`, eg: `reverse('freeradius:authorize')`